tests/remote/basic/suite.rc

```diff
@@ -7,14 +7,8 @@
 # determined by the [job platforms] section in site/user config files.
 [scheduling]
     [[graph]]
-        R1 = "foo => bar"
+        R1 = foo
```

Comment:
There used to be two tasks one testing remote host and the other remote owner functionality. Since we've removed the latter I removed the task with it.